### PR TITLE
Add node-enricher to komodor-agent chart

### PR DESCRIPTION
**Pull request by rannyeli**
public: Install node-enricher service as part of the komodor-agent daemonset

### CODE REVIEW

**Comment by rannyeli** [comment]
Task linked: [CU-860t053vx [Agent] Implement a new event listener for spot interruption events](https://app.clickup.com/t/860t053vx)

File: charts/komodor-agent/values.yaml [hunk 1]
```diff
@@ -54,6 +54,8 @@ capabilities:
   metrics: true
   # capabilities.networkMapper -- (bool) Enable network mapping capabilities by the komodor agent
   networkMapper: true
+  # capabilities.nodeEnricher -- (bool) Enable node enricher capabilities by the komodor agent
+  nodeEnricher: true
```
Comment (MickaelAlliel) [inline charts/komodor-agent/values.yaml:58]: I'd start by disabling it by default, testing on our own clusters for a few days and only then enabling it for customers. We do have the new agent release process which should validate it enough before the actual release but sync with @Moran-k to make sure how it works.

File: charts/komodor-agent/templates/node-enricher/_containers.tpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{{- define "node_enricher.daemonset.container" }}
+{{- if .Values.capabilities.nodeEnricher }}
```
Comment (MickaelAlliel) [inline charts/komodor-agent/templates/node-enricher/_containers.tpl:2]: No probes?

**Comment by MickaelAlliel** [comment]
Run `make generate-readme` :)

**Pull request merged**